### PR TITLE
Adjust zindex header to avoid it behind over the menu on mobile

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The header-toolbar is over the opened menu on mobile
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Build admin-dev/themes/new-theme assets and go on Catalog -> Products on mobile, open the menu and the menu should be over everything

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17459)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

File: admin-dev/themes/new-theme/scss/components/layout/_header_toolbar.scss [hunk 1]
```diff
@@ -82,8 +80,7 @@
     position: relative;
 
     .notification-container {
-      @include notification_container(1.5rem)
-      position: absolute;
+      @include notification_container(1.5rem) position: absolute;
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/scss/components/layout/_header_toolbar.scss]: Why remove the EOL after the include ?

Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/components/layout/_header_toolbar.scss]: God, that should be a prettier misconfiguration, I've to fix that :)

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by SD1982** [comment]
This pull request correctly fix the described issue.

**Before PR :** 
#PR17459 Adjust zindex header to avoid it behind over the menu on mobile before PR

**After PR :**
#PR17459 Adjust zindex header to avoid it behind over the menu on mobile after PR png

Thanks @NeOMakinG !

**Comment by Progi1984** [comment]
Thanks @NeOMakinG

**Pull request merged**